src/main/java/org/infernalstudios/shieldexp/init/ShieldDataLoader.java

```diff
@@ -29,8 +29,10 @@
 public class ShieldDataLoader extends SimpleJsonResourceReloadListener {
     public static final Gson GSON = (new GsonBuilder()).setPrettyPrinting().disableHtmlEscaping().create();
     public static Map<ResourceLocation, JsonElement> FILE_MAP = new HashMap<>();
-    public static Map<String, Map<String, Double>> SHIELD_STATS = new HashMap<>();
-    public static List<Map.Entry<ResourceLocation, JsonElement>> toSync = new ArrayList<>();
+    public static final Map<String, Map<String, Double>> SHIELD_STATS = new ShieldStatsMap();
+    public static final List<Map.Entry<ResourceLocation, JsonElement>> toSync = new ArrayList<>();
+
+    public static final String DEFAULT_SHIELD_NAME = ShieldExpansion.MOD_ID + ":default";
```

Comment:
Wouldn't it be better to do this, instead of converting the RLs to Strings every time?

```suggestion
    public static final Map<ResouceLocation, Map<String, Double>> SHIELD_STATS = new ShieldStatsMap();
    public static final List<Map.Entry<ResourceLocation, JsonElement>> toSync = new ArrayList<>();

    public static final ResourceLocation DEFAULT_SHIELD_NAME = new ResourceLocation(ShieldExpansion.MOD_ID, "default");
```

Reply:
That is if you want my PR to change the semantics around `SHIELD_STATS`, which I have no problem doing. I just typically try to only add what's needed rather than changes what's existing.

Reply:
Since your PR already changes some aspect of this, I don't see a reason to change this semantic as well.